src/detect-engine-alert.c

```diff
@@ -421,6 +433,11 @@ static inline void PacketAlertFinalizeProcessQueue(
                 SCLogDebug("sid:%u: is a pass rule, so break out of loop", s->id);
                 break;
             }
+
+            // TODO we can also drop if alert is suppressed, right?
```

Comment:
need a test for this:
ips sig that drops, alert is suppressed. Overrides a app_filter accept